internal/controller/bucket_controller_test.go

```diff
@@ -481,7 +481,7 @@ func TestBucketReconciler_reconcileSource_generic(t *testing.T) {
 			wantErr:     true,
 			assertIndex: index.NewDigester(),
 			assertConditions: []metav1.Condition{
-				*conditions.TrueCondition(sourcev1.FetchFailedCondition, sourcev1.AuthenticationFailedReason, "failed to get secret '/dummy': secrets \"dummy\" not found"),
+				*conditions.TrueCondition(sourcev1.FetchFailedCondition, sourcev1.AuthenticationFailedReason, "failed to get secret"),
```

Comment:
Looking the modified tests here, we have introduced a regression. We no longer tell users which secret are we looking for. All the errors should contain the secret name, so people will have a chance to debug this. I guess we have the same issue for OCIRepository.